repos.d/izzyondroid.yaml

```diff
@@ -0,0 +1,26 @@
+###########################################################################
+# IzzyOnDroid andriod repository
+###########################################################################
+- name: izzyondroid
+  type: repository
+  desc: IzzyOnDroid F-Droid Repository
+  family: fdroid
```

Comment:
This should have different family, as doesn't (seem to) have common packages with f-droid.